apps/artist/routes.coffee

```diff
@@ -69,7 +70,7 @@ sd = require('sharify').data
   req.user.followArtist req.params.id,
     error: res.backboneError
     success: ->
-      res.redirect "/artist/#{req.params.id}"
+      res.redirect "/artist/#{req.params.id}?#{qs.stringify req.query}"
```

Comment:
There are where we just pass along any current query params, from /payoff and /follow